devops/aws/roles/rpc/tasks/main.yml

```diff
@@ -41,10 +41,13 @@
   # Check for the status every 100s
   delay: 100
 
-- name: Run query node containers
-  command: yarn workspace query-node-root start:dev
+- name: Run service containers
+  command: yarn start
```

Comment:
yarn start will also try to start joystream-node service which will try to listen on same port 9944 as node started with systemd.